projects/vdk-core/src/vdk/internal/builtin_plugins/builtin_hook_impl.py

```diff
@@ -130,6 +131,7 @@ def vdk_start(plugin_registry: PluginRegistry, command_line_args: List) -> None:
     plugin_registry.add_hook_specs(JobRunHookSpecs)
     plugin_registry.load_plugin_with_hooks_impl(JobConfigIniPlugin())
     plugin_registry.load_plugin_with_hooks_impl(TerminationMessageWriterPlugin())
+    plugin_registry.load_plugin_with_hooks_impl(JobRunSummaryOutputPlugin())
```

Comment:
Are we outputting the result to a file for every job? It's ok if we are, but we should probably document the behaviour.

Reply:
Only if enabled (summary output location is set).